### ISSUE TITLE
Console Log Error when following a link in timeseries/properties section.

### ISSUE THREAD

**Issue by ppeyro** [predix-ui/predix-ui.github.io#10]
In timeseries -> scroll down to xaxisconfig (or yaxisconfig) properties.
Follow the link  'https://github.com/PredixDev/px-vis'
redirects to a blank page with the following error in the console log:

Refused to display 'https://github.com/PredixDev/px-vis' in a frame because an ancestor violates the following Content Security Policy directive: "frame-ancestors 'none'".

Screenshot attached:
![screen shot 2016-11-18 at 1 57 53 pm](https://cloud.githubusercontent.com/assets/10835611/20448102/2e0ab8b4-ad97-11e6-8c79-b326ac8b8fd9.png)

**Comment by davidrleonard** [comment]
@ppeyro I can't reproduce this bug, perhaps a recent change fixed it. Are you still seeing this issue?

**Comment by ppeyro** [comment]
@davidrleonard Yes, this one was fixed a while ago, we just didn't close it, maybe?  Closing it now.

**Comment by ppeyro** [comment]
This one is back again....

Following the link to github, causes the console log error and a blank page view.
Today's screenshot:

![screen shot 2017-03-20 at 12 51 01 pm](https://cloud.githubusercontent.com/assets/10835611/24119453/7bed3a3e-0d6e-11e7-94c0-7449bd0a248e.png)

/cc @davidrleonard

**Comment by nonmetalhail** [comment]
This still exists in the redesign

**Comment by nonmetalhail** [comment]
It also keeps logging. Not just once
![screen shot 2017-08-01 at 5 37 06 pm](https://user-images.githubusercontent.com/1649404/28852720-287eb798-76e0-11e7-86aa-6e9cdb0e0d87.png)
